driver/sources/src/main/scala/org/virtuslab/ideprobe/config/PathsConfig.scala

```diff
@@ -0,0 +1,15 @@
+package org.virtuslab.ideprobe.config
+
+import java.nio.file.Path
+
+case class PathsConfig(base: Option[Path],
+                       instances: Option[Path],
+                       workspaces: Option[Path],
+                       screenshots: Option[Path],
+                       cache: Option[Path])
+
+object PathsConfig {
+  def apply(): PathsConfig = {
```

Comment:
Just put default parameters in the case class, or instead of `apply` create a Default val. But probably none of these is needed.